frontend/src/app/services.ts

```diff
@@ -240,6 +240,26 @@ const mutationApi = hitasApi.injectEndpoints({
                 return [];
             },
         }),
+        removeApartment: builder.mutation<
+            IApartmentDetails,
+            {
+                data?: IApartmentWritable;
+                id: string | undefined;
+                housingCompanyId: string;
+            }
+        >({
+            query: ({data, id, housingCompanyId}) => ({
+                url: `housing-companies/${housingCompanyId}/apartments/${id}`,
+                method: "DELETE",
+                body: data,
```

Comment:
This line can be completely removed from this request here, few lines above and where this function is called. `data` doesn't need to be sent to the API at all when removing an object :)

Reply:
Ah - I see! I understood from the earlier comment that it was supposed to be a non-required part of the request. Now removed entirely from both the mutation function in `services.js` and the function call in `ApartmentCreatePage.tsx`